modules/wechat_qrcode/samples/qrcode.py

```diff
@@ -0,0 +1,60 @@
+# This file is part of OpenCV project.
+# It is subject to the license terms in the LICENSE file found in the top-level directory
+# of this distribution and at http://opencv.org/license.html.
+#
+# Tencent is pleased to support the open source community by making WeChat QRCode available.
+# Copyright (C) 2020 THL A29 Limited, a Tencent company. All rights reserved.
```

Comment:
Sample files in OpenCV don't include license headers (they are often used in documentation).